clients/drcachesim/simulator/caching_device_stats.h

```diff
@@ -245,6 +247,9 @@ class caching_device_stats_t {
 
     int64_t num_inclusive_invalidates_;
     int64_t num_coherence_invalidates_;
+    // TODO: Hmm, maybe ought to combine inclusive and exclusive invalidation
```

Comment:
Style: Always point at an issue with a TODO: `TODO i#5995:`.

Reply:
Right, that was just a note to self that I forgot to delete.